src/products/utils/data.ts

```diff
@@ -210,6 +211,7 @@ export function getProductUpdatePageFormData(
     name: maybe(() => product.name, ""),
     privateMetadata: product?.privateMetadata?.map(mapMetadataItemToInput),
     publicationDate: maybe(() => product.publicationDate, ""),
+    rating: maybe(() => product.rating, null),
```

Comment:
We don't want to use `maybe` anymore, it was used when optional chaining wasn't available.

Reply:
done